src/forms/reset-password-popup/reset-password/index.jsx

```diff
@@ -161,6 +161,7 @@ const ResetPasswordPage = () => {
   } if (status === PASSWORD_RESET_ERROR || status === PASSWORD_RESET.INVALID_TOKEN) {
     dispatch(setCurrentOpenedForm(FORGOT_PASSWORD_FORM));
   } else if (status === SUCCESS) {
+    trackPasswordResetSuccess();
```

Comment:
Do we have reset password success event in authn? I am not sure about this.

Reply:
nope, we don't have.

but we have a 'Reset password success rate` metric. For this metric, we have to add this event.